lib/cloud_controller/metrics/prometheus_updater.rb

```diff
@@ -37,20 +56,17 @@ def update_summary_metric(metric, value, message)
     end
 
     def update_deploying_count(deploying_count)
-      update_gauge_metric(:cc_deployments_deploying, deploying_count, 'Number of in progress deployments')
+      update_gauge_metric(:cc_deployments_in_progress_total, deploying_count, 'Number of in progress deployments')
     end
 
     def update_user_count(user_count)
-      update_gauge_metric(:cc_total_users, user_count, 'Number of users')
+      update_gauge_metric(:cc_users_total, user_count, 'Number of users')
     end
 
-    def update_job_queue_length(pending_job_count_by_queue, total)
+    def update_job_queue_length(pending_job_count_by_queue)
       pending_job_count_by_queue.each do |key, value|
-        metric_key = :"cc_job_queue_length_#{key.to_s.underscore}"
-        update_gauge_metric(metric_key, value, docstring: "Job queue length for worker #{key}")
+        update_gauge_metric(:cc_job_queues_length_total, value, "Job queue length for worker #{key}", labels: { queue: key.to_s.underscore })
```

Comment:
No it doesn't. I also don't like that we have to pass the docstring all the time when emitting metrics. I refactored that so that all metrics get registered on creation of the PrometheusUpdater.